examples_utils/benchmarks/platform_assessment/run_paperspace_tests.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2023 Graphcore Ltd. All rights reserved.
+#!/usr/bin/env -S python3 -u
+import argparse
+import datetime
+import os
+
+
+def upload_report(api_key, folder, dataset, version):
+    """Upload reports to gradient"""
+    os.system(f"gradient apiKey ${api_key}")
+    # for file in `find ${5} -name "*.tar.gz"`
+    for file in os.listdir(folder):
+        if "tar.gz" in file:
+            os.system(
+                f"gradient datasets files put \
+                      --id ${dataset}:${version} --source-path ${file}"
+            )
+            os.system(
+                f"gradient datasets versions commit \
+                      --id ${dataset}:${version}"
+            )
```

Comment:
This function is never used, I think you could remove it. We'll need to implement a version that works.